docs/releases/5.2.md

```diff
@@ -219,6 +219,99 @@ If using custom styling for the breadcrumbs, this class has changed from singula
 | ---------------- | ----------------- |
 | `'w-breadcrumb'` | `'w-breadcrumbs'` |
 
+### Snippets templates refactored to reuse the shared `slim_header.html` template
+
+The templates for the snippets views have been refactored to reuse the shared `slim_header.html` template. If you have customised or extended the templates, e.g. for [](wagtailsnippets_custom_admin_views), you will need to update them to match the new structure. As a result, the following templates have been removed:
+
+- `wagtailsnippets/snippets/headers/_base_header.html`
+- `wagtailsnippets/snippets/headers/create_header.html`
+- `wagtailsnippets/snippets/headers/edit_header.html`
+- `wagtailsnippets/snippets/headers/history_header.html`
+- `wagtailsnippets/snippets/headers/list_header.html`
+- `wagtailsnippets/snippets/headers/usage_header.html`
+
+In most cases, the usage of those templates can be replaced with the `wagtailadmin/shared/headers/slim_header.html` template. Refer to the snippets views and templates code for more details.
+
+### `BaseSidePanels`, `PageSidePanels` and `SnippetSidePanels` classes are removed
```

Comment:
I wonder how we draw the line between "undocumented internals" vs "Wagtail customisations". One can argue these classes can be considered "undocumented internals"...?

Reply:
Yep, I think it's quite likely that this can be moved into the "undocumented" section too. I didn't want to get too derailed into checking the documentation status of everything in the list, so as a first pass I just moved the items that specifically mentioned being undocumented.